service/src/main/resources/quips/complibot/winsults.json

```diff
@@ -21,7 +27,10 @@
     "ssml":"I can see by looking at you that you don't get much sleep, since you have to wake up early every day to look so great.",
     "usageRules":[
       
-    ]
+    ],
+    "tags":[],
+    "targetableText":"I can see that [TARGET] doesn't get much sleep, since you'd have to wake up early every day to look that great!",
```

Comment:
I don't like the way this translated.  I realize that it's "you" in the sense of "a person", but it still sounds awkward.